### DISCUSSION TITLE
Renaming the Repo for Branding issues.

### DISCUSSION THREAD

**Discussion by sca075** [sca075/mqtt_vacuum_camera#187] [Announcements]
As you have probably noticed the repo name changed, and soon I would need also to rename the camera domain in HA.
I will try to keep your installations working without big braking changes, there is already some idea on how to do that.
The custom component development will continue in regular bases. 
About HACS, there will be no changes for the time be, so you might receive the updates without issue.
The only change for the moment is for the contributors. If you start to contribute now there could be some technical issue, that is possible to solve following the GitHub guides.

**Comment by sca075** [comment]
The new logo was approved (still need to create and upload a dark version of it). Working on the migration of the home assistant domain (that will be quite a complex process), will try to make it as painless possible (by automating it), this will take a while, as we need to be 300% sure that there will be no problems for anyone of you.

**Reply by Drafteed** [reply]
Please consider to push a new logo to https://github.com/home-assistant/brands

**Reply by Drafteed** [reply]
Oops, sorry, saw that you already added it 👍

**Reply by sca075** [reply]
@Drafteed  we are almost ready and the next hours will publish the new 2024.06.4 that will rename the camera to mqtt_vacuum_camera. It will be quite easy to move it, just an additional reboot to do after installation and a new backup of Home Assistant. The new day and night version of the logs are going to be in the fronted as soon mqtt_vacuum_camera will be setup.
If you are one our users and you like this component, please consider to start the project, this helps quite a lot ;)

**Comment by sca075** [comment]
Okay.. first test completed, there are some adjustments to do but it looks really promising. The issue we need to solve is to not generate a second camera but probably this happened as the old folder with the code we did run was still in there.

### In summary:
* We managed to get all options as it was on the new mqtt_vacuum_camera.
* The procedure will require a new restart of HA. So in total two restarts when it goes right.
* We will re-test it but apparently, the transfer to the new domain could be quite easy 👍

**Comment by sca075** [comment]
Everyone, the 2024.06.4 Camera version had as scope to migrate the Valetudo Vacuum Camera to MQTT Vacuum Camera. 
I hope this was quite easy due to the effort made in order to migrate to the new folder the integration. 
In the next release the component will be renamed for good, meaning that, if the next version should be (if all goes like planned) installed on the mqtt_vacuum_camera folder. 
I don't know, not sure how many of you still need to migrate, so I'm working on a possible solution (similar to what implemented) so that the repo and the installation folder of the camera will match in the future (currently both can be reach as per the camera update should be available to do not create discomforts. 

I really wish that the over 200 installations went smoothly as usual, so far no reported failures. In this period I'm limiting the modifications to the code, but soon we will need to restart to work on it to improve it mainly.

**Comment by sca075** [comment]
We are close to release the 2024.07.0 that is fist MQTT Vacuum Camera fully renamed version of the former Valetudo Vacuum Camera. The transition to the new Domain looks went quite good with over 340 downloads we got in this days. Still few of you can download considering all, anyhow from the next version will be probably necessary to update HACS to the new repo mqtt_vacuum_camera, also if HACS is taking note of the changes on the repo name.

Hopefully after this long month without any significant updates of the camera, we can now continue to work on improve the code and perhaps also the functionality of the Camera.
